core/model/modx/processors/security/group/getnodes.class.php

```diff
@@ -131,7 +131,7 @@ public function prepareGroup(modUserGroup $group) {
         $c->limit(1);
         $count = $this->modx->getCount('modUserGroup', $c);
         return array(
-            'text' => $group->get('name').' ('.$group->get('id').')',
+            'text' => htmlentities($group->get('name'), ENT_QUOTES, 'UTF-8') . ' ('.$group->get('id') . ')',
```

Comment:
That's a fair point. In most of my projects I tend to just hardcode UTF-8 because it takes me a lot of effort to come up with a reason why it wouldn't be UTF-8, but I'd be happy to make it read from the setting as well. I guess that kinda is the way it's done in the rest in MODX, so should be done here too.